packages/react-tooltip/package.json

```diff
@@ -44,7 +42,7 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/react-make-styles": "9.0.0-beta.4",
+    "@griffel/react": "1.0.0",
```

Comment:
nit: This should go after the `@fluentui/*` packages to maintain alphabetical order.